packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/commitment/commitment.routing.js

```diff
@@ -11,8 +11,13 @@ export default /* @ngInject */ ($stateProvider) => {
       serviceId: /* @ngInject */ (serviceInfos) => serviceInfos.serviceId,
       me: /* @ngInject */ (currentUser) => currentUser,
       trackingPrefix: () => 'dedicated::server::dashboard',
+      pageTrackingPrefix: () =>
+        'dedicated::dedicated-server::server::dashboard',
```

Comment:
**question:** Couldn't `trackingPrefix` been reused instead?

Reply:
@antleblanc 
No. It's not same for page tracking and click tracking in this case.